data/nm/retired/Michelle-Lujan-Grisham-cb607080-ff7b-48c8-acf4-0180e6e2f999.yml

```diff
@@ -11,6 +11,10 @@ roles:
   end_date: '2023-01-01'
   type: governor
   jurisdiction: ocd-jurisdiction/country:us/state:nm/government
+offices:
+- classification: capitol
+  address: 490 Old Santa Fe Trail Room 400; Santa Fe, NM 87501
+  voice: 505-476-2200
 links:
```

Comment:
@sroomf move out of retired, to executive maybe?

Reply:
We dont track executives for NM so I will keep this person in retired